libs/utils/src/namespaces/string.ts

```
@@ -60,6 +60,19 @@ export const parseLayoutLocator = (payload: SortablePayload | null): LayoutLocat
 export const sanitize = (html: string, options?: sanitizeHtml.IOptions) => {
   return sanitizeHtml(html, {
     ...options,
+    allowedTags: [
+      ...options?.allowedTags,
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)